tests/unit/test_utils.py

```diff
@@ -249,8 +249,7 @@ def test_rmtree_errorhandler_reraises_error(tmpdir: Path) -> None:
     # Create directory without read permission
     subdir_path = tmpdir / "subdir"
     subdir_path.mkdir()
-    path = str(subdir_path)
-    os.chmod(path, stat.S_IWRITE)
+    os.chmod(subdir_path, stat.S_IWRITE)
```

Comment:
Did ruff do this, or was it a manual change? I'm very concerned if this was a ruff change. If it was manual, it's unrelated to this PR, isn't it?